ch03/02_bonus_efficient-multihead-attention/mha-implementations.ipynb

```diff
@@ -244,7 +244,7 @@
     "        qkv = self.qkv(x)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.            self.register_buffer(
</pre><p>There is no need anymore for the "mask" buffer, isn't it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rasbt/LLMs-from-scratch/pull/131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Yes, good catch, that should be redundant.